src/main/resources/decodes/resources/comprun_en_US.properties

```diff
@@ -15,6 +15,7 @@ RunComputationsFrame.saveOutputButton = Save Output Data
 RunComputationsFrame.timeLabel = Time
 RunComputationsFrame.noCompSelectedErr = No computations selected to run!
 RunComputationsFrame.saveCompOutput = Save Computation Output?
+RunComputationsFrame.cancelComputationExecution = Cancel Computation Execution?
```

Comment:
@ktarbet if you can paste the appropriate Spanish for this as a reply I can add it to the es_US.properties file.